packages/next/src/server/config-schema.ts

```diff
@@ -138,14 +150,10 @@ const zTurbopackRuleConfigItem: zod.ZodType<TurbopackRuleConfigItem> = z.union([
   ),
   zTurbopackRuleConfigItemOptions,
 ])
+
 const zTurbopackRuleConfigItemOrShortcut: zod.ZodType<TurbopackRuleConfigItemOrShortcut> =
   z.union([z.array(zTurbopackLoaderItem), zTurbopackRuleConfigItem])
 
-const zTurbopackCondition: zod.ZodType<TurbopackRuleCondition> = z.object({
-  path: z.union([z.string(), z.instanceof(RegExp)]).optional(),
-  content: z.instanceof(RegExp).optional(),
-})
-
 const zTurbopackConfig: zod.ZodType<TurbopackOptions> = z.strictObject({
   rules: z.record(z.string(), zTurbopackRuleConfigItemOrShortcut).optional(),
   conditions: z.record(z.string(), zTurbopackCondition).optional(),
```

Comment:
Remove?

Reply:
Same as above, I'm planning on removing this in a follow-up PR.